Freshli.Web.Worker/appsettings.Development.json

```diff
@@ -0,0 +1,9 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
```

Comment:
Should `Hangfire` be included here as well?

Reply:
I'm not sure. I added it just in case in e530c5d.